### PR TITLE
Feat users index

### PR DESCRIPTION
**Pull request by kirouackp0**
### Ticket Link
(Found on Jira in your ticket description)
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-12

### User Story
(Linked to on Jira in your ticket description)
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-12

### What does this PR do?
Adds back and front to index of users (displays users in cards with a photo attached). This is the MVP version and will still be improved. Adds premium filter for dating mode to be deactivated for non premium users next week. 

I removed the gap from the light-theme.css file as well as it was causing issues with other pages. 

Please note that a future PR will resolve the following:
- ensure that current_user is not displayed in index
- add filter for user_interests
- make the profile photo a button to user_show page

### Where should the reviewer start?
Launch server and make sure the front looks like the screen shot. 

### How should this be manually tested?
Check that the full array of users from seeds appears in index.

### Screenshots (if appropriate)
Screen Shot 2023-09-07 at 13 16 39
Screen Shot 2023-09-07 at 09 59 51

### Are there any breaking changes?
There shouldn't be. it works for me

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,6 +4,11 @@
   resources :chat_users
   resources :user_interests
   resources :interests
+  resources :users do
+    collection do
+      get 'filter'
```
Comment (Creiwry) [inline config/routes.rb:9]: This change has made the tests fail. Please run the tests and see why. If you have trouble with it, let me know.

**Review by Creiwry — CHANGES_REQUESTED** [review]
Please look at how you set up routes for users. This is making the tests fail when it shouldn't!

File: config/routes.rb (hunk 1 above)
Comment (kirouackp0) [reply config/routes.rb:9]: On it

**Review by Creiwry — APPROVED** [review]
Approved, with the note that the filter functionality will be finished in a future PR

**Pull request merged**